### ISSUE TITLE
Not working with laravel 8

### ISSUE THREAD

**Issue by PauBenetPrat** [GlueDev/laravel-stackdriver#26]
Hi, could you update package to support laravel 8?

Or if you prefer, I can submit a PR.

Thanks,
Pau

**Comment by bhaidar** [comment]
Hi @PauBenetPrat 
Why wouldn't this package work with Laravel 8? What are the things to fix in order to allow it to work with Laravel8? Thanks

**Comment by LarryBarker** [comment]
@PauBenetPrat What is the issue? We're looking to use with Laravel 8 so it would be helpful to know what issues you're having

**Comment by PauBenetPrat** [comment]
Hi, sorry for the delay, problem resides on opencensus/opencensus. If you create  a new laravel projecte and composer require gluedev/laravel-stackdriver it returns following error:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - opencensus/opencensus v0.5.2 requires ramsey/uuid ~3 -> found ramsey/uuid[3.0.0-alpha1, ..., 3.x-dev] but the package is fixed to 4.1.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - gluedev/laravel-stackdriver 0.2.2 requires opencensus/opencensus ^0.5.2 -> satisfiable by opencensus/opencensus[v0.5.2].
    - Root composer.json requires gluedev/laravel-stackdriver ^0.2.2 -> satisfiable by gluedev/laravel-stackdriver[0.2.2].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
```

Opencensus requires ramsey/uuid 3.9 but laravel 8 do require 4.0^ ...

**Comment by andrewmclagan** [comment]
Also seeing this. on 8.x
